### PR TITLE
fixed code format in WebAppGenerator.java

### PR DESCRIPTION
**Pull request by ShivangMishra**
## Description

Fixes # (issue)

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [ ] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [ ] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] My code follows the style guidelines of this project
 - [ ] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [ ] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [ ] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java [hunk 1]
```diff
@@ -14,33 +14,25 @@
 package org.eclipse.jkube.generator.webapp;
 
 import java.io.File;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```
Comment (rohanKanojia) [inline jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java:17]: Have you set up your IDE as per the steps provided in https://www.eclipse.org/jkube/contributing/#project-setup ? 

Usually, we avoid wildcard imports.

**Comment by ShivangMishra** [comment]
@rohanKanojia can you please merge this branch into master while I finalize #2244 ?

File: jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java (hunk 1 above)
Comment (ShivangMishra) [reply jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java:17]: > Have you set up your IDE as per the steps provided in https://www.eclipse.org/jkube/contributing/#project-setup ?
> 
> Usually, we avoid wildcard imports.

image

This is my configuration. I followed the guide...

File: jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java [hunk 2]
```diff
@@ -232,4 +191,35 @@ private String getDeploymentDir(AppServerHandler handler) {
   private String getUser(AppServerHandler handler) {
     return getConfig(Config.USER, handler.getUser());
   }
+
```
Comment (sunix) [inline jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java:194]: hello any reason why this part is moving at the end? I am not sure the formatter is enforcing it

File: jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java (hunk 1 above)
Comment (anurag-rajawat) [reply jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java:17]: Hi @ShivangMishra!
You can fix wildcard imports by following https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

**Comment by manusa** [comment]
I'm going to close this one since it's not really adding any value and it's still not ready.

**Pull request closed without merging**